packages/amplify-category-notifications/src/validate-filepath.ts

```diff
@@ -0,0 +1,14 @@
+import * as fs from 'fs-extra';
+
+/**
+ * Validate the given filePath
+ * @param filePath - path to file
+ * @returns true on success
+ */
+export const validateFilePath = (filePath: string): boolean|string => {
```

Comment:
Is this function needed , We can just check ```fs.existsSync``` directly

Reply:
this function is passed to the inquirer questions object in `apns-cert-config.ts` and `apns-key-config.ts`, leaving as is in this effort to move to TS